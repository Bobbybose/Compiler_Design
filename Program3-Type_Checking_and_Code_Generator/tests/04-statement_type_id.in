int a;
float b;
char c;
a = 2;
b = 2.1;
c= 'a';