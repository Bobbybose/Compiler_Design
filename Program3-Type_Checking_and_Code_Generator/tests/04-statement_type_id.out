Code:
%t1 = 2
a = %t1
%t2 = 2.100000
b = %t2
%t3 = 'a'
c = %t3

